### ISSUE TITLE
PHP FPM client limit under debian apache is by default fairly limited 

### ISSUE THREAD

**Issue by hayden-t** [virtualmin/virtualmin-gpl#680]
[mpm_prefork:error] [pid 119747] (11)Resource temporarily unavailable: AH00159: fork: Unable to fork new process

Caused by systemctl default task limit of 614

https://unix.stackexchange.com/questions/759958/apache-mpm-prefork-limited-to-306-connections-yet-server-limit-set-to-1000-ah0

**Comment by hayden-t** [comment]
not really, there are many system settings that are set-table via vmin/webmin, there needs to be at least a note about it in the mpm prefork config file, as it was very difficult and obscure for me to work this out.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.
